src/main/yamcs/mdb/oresat.xml

```diff
@@ -286,6 +286,16 @@
 
                 </EntryList>
             </SequenceContainer>
+            <SequenceContainer name="Satnogs_Packet">
```

Comment:
In general, don't modify the XTCE because this file is autogenerated.

We should probably be adding it to our `.gitignore`, so it doesn't cause confusion.